README.md

```diff
@@ -11,20 +11,28 @@
-
-Rook turns storage software into self-managing, self-scaling, and self-healing storage services. It does this by automating deployment, bootstrapping, configuration, provisioning, scaling, upgrading, migration, disaster recovery, monitoring, and resource management. Rook uses the facilities provided by the underlying cloud-native container management, scheduling and orchestration platform to perform its duties.
+Rook is an open source **cloud-native storage orchestrator** for Kubernetes,
+providing the platform, framework, and support for [Ceph storage](https://docs.ceph.com/en/latest/) to natively integrate with Kubernetes.
 
+Rook automates deployment and management of Ceph to provide self-managing, self-scaling, and self-healing storage services.
+The Rook operator does this by building on Kubernetes resources to deploy, configure, provision, scale, upgrade, monitor, and monitor Ceph.
 Rook integrates deeply into cloud native environments leveraging extension points and providing a seamless experience for scheduling, lifecycle management, resource management, security, monitoring, and user experience.
 
-For more details about the storage solutions currently supported by Rook, please refer to the [project status section](#project-status) below.
-We plan to continue adding support for other storage systems and environments based on community demand and engagement in future releases. See our [roadmap](ROADMAP.md) for more details.
-
 Rook is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CNCF) as a [graduated](https://www.cncf.io/announcements/2020/10/07/cloud-native-computing-foundation-announces-rook-graduation/) level project. If you are a company that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details about who's involved and how Rook plays a role, read the CNCF [announcement](https://www.cncf.io/blog/2018/01/29/cncf-host-rook-project-cloud-native-storage-capabilities).
 
+## Ceph Provider
+
+The status of the Ceph storage provider is **Stable**.
+Features and improvements will be planned for many future versions. Upgrading between versions will be provided
+to ensure backward compatibility between releases.
+
+| Description                                                                                                                                                    | API Group       | Status |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------- | ------ |
+| [Ceph](https://ceph.com/) is a distributed storage system that provides file, block and object storage and is deployed in large scale production clusters. | ceph.rook.io/v1 | Stable |
```

Comment:
How about removing the table completely and moving the information out of it?

Reply:
Removed this section and integrated a paragraph on the Stable storage with the main section